composer.json

```diff
@@ -1,15 +1,15 @@
 {
-    "name": "vaimo/webdriver-binary-downloader",
+    "name": "bolt/webdriver-binary-downloader",
     "type": "library",
     "license": "MIT",
     "description": "General-issue library that enables building any kind of WebDriver binary downloader",
     "require": {
-        "php": ">=5.3.0",
-        "composer/composer": "^1.0",
-        "composer-plugin-api": "^1.0"
+        "php": ">=7.2.0",
```

Comment:
In our case, we're sticking with Symfony's requirements. Makes it easier for us, but your usecase might vary.

Reply:
I agree, we should be supporting 7.2+ because everything else is EOL. Although I'm realizing this PR is a bit wonky because its comparing against the previous state of the module, and I think most of the changes you've made are also ones made in the most recent version of the fork.. however the PR doesn't show that. hrmmmm.